trio/_core/_windows_cffi.py

```diff
@@ -327,13 +516,14 @@ def raise_winerror(
     filename: str | None = None,
     filename2: str | None = None,
 ) -> NoReturn:
+    # assert sys.platform == "win32"  # TODO: make this work in MyPy
     if winerror is None:
-        err = ffi.getwinerror()
+        err = ffi.getwinerror()  # type: ignore[attr-defined,unused-ignore]
```

Comment:
May want a comment in this func to explain that `getwinerror()` is only defined on windows, hence the type-ignore.